lib/ansible/modules/net_tools/ldap/_ldap_attr.py

```diff
@@ -255,11 +257,28 @@ def main():
         module.fail_json(msg=missing_required_lib('python-ldap'),
                          exception=LDAP_IMP_ERR)
 
-    # Update module parameters with user's parameters if defined
-    if 'params' in module.params and isinstance(module.params['params'], dict):
+    # For Ansible-2.9.x and below, allow the params module parameter with a warning
+    if LooseVersion(module.ansible_version) < LooseVersion('2.10'):
```

Comment:
Is this so the same code can be backported to previous Ansible releases?

Reply:
Correct. we removed params from yum_repository and jenkins_plugin on a short cycle before but didn't backport the changes.  Since this is a security issue, we should backport it.  I tried to make the change do less for 2.9 and earlier (only disallowing bind_pw usage).